k/kong/2.3.4/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM registry.access.redhat.com/ubi7/ubi AS build
+
+WORKDIR /root
+
+ENV VERSION=2.3.4
+ENV HOME=/root
+ENV OPENSSL_DIR=/usr/local/openssl
+ENV LD_LIBRARY_PATH=/usr/local/lib
+ENV PATH=$HOME/kong/bin:/usr/local/openresty/bin:/usr/local/openresty/nginx/sbin:/usr/local/luarocks/bin:$PATH
+
+RUN yum -y install libtool zlib zlib-devel unzip zip make wget git patch make gcc-c++ ncurses-devel
+
+RUN git clone https://github.com/yaml/libyaml.git \
+    && cd libyaml \
+    && git checkout tags/0.1.4 \
+    && ./bootstrap && ./configure && make && make install;
+
+RUN wget ftp://ftp.gnu.org/gnu/readline/readline-7.0.tar.gz \
+    && tar -xzf readline-7.0.tar.gz \
+    && cd readline-7.0 \
+    && ./configure \
+    && make \
+    && make install \
+    && cd .. ;
+
+RUN wget https://www.lua.org/ftp/lua-5.1.5.tar.gz \
+    && tar -xzvf lua-5.1.5.tar.gz \
+    && cd  lua-5.1.5 \
+    && make linux \
+    && cd .. ;
+
+RUN git clone https://github.com/kong/openresty-build-tools \
+    && cd openresty-build-tools \
+    && lineNum=$(grep -n "tar -xzvf openresty-\$OPENRESTY_VER.tar.gz" kong-ngx-build | tr -s " " " " |cut -d: -f 1) \
+    && lineNum=`expr $lineNum + 1` \
+    && sed -i  -e ''$lineNum'icd -' kong-ngx-build \
+    && sed -i  -e ''$lineNum'igit checkout 2763a421d6219c8cb2bbd39246de619dc796bab6' kong-ngx-build \
+    && sed -i  -e ''$lineNum'icd openresty-${OPENRESTY_VER}/bundle/LuaJIT-2.1-20190507' kong-ngx-build \
```

Comment:
kong 2.3.4 seems to be from May 2022, https://github.com/kong/openresty-build-tools seems to have been archived in Dec. 2019, so something here seems a bit odd, have you looked at https://github.com/Kong/kong-build-tools by any chance? that seems to be the replacement.

The reason I started looking at it was the luajit version here seemed to be pretty old (from 20190